common/character_interactions/00_marriage_interactions.txt

```diff
@@ -73,6 +73,11 @@ arrange_marriage_interaction = {
 				limit = {
```

Comment:
> You can select Kael'thas in Anasterian court, but then it redirects recipient to Kael'thas, breaking character clearing.

You always could. Except it would redirect to Antonidas.

Reply:
> Also, it would be nice to give governments that can't be matchmakers for their courtiers a tooltip about it, like these.

Added in 4e077e557663f63a80e35943b1819015b9628525.